src/style/components/group-video-grid.less

```diff
@@ -163,9 +163,31 @@
         object-fit: cover;
       }
 
+      &__action_icon {
```

Comment:
We actually would like to get rid of less, maybe we really should stopping adding any new less code to finally force us to refactor :/

Reply:
I don't think we need the code, so let's get rid of it.